src/main/resources/scripts/anomalyDetectionTesting.txt

```diff
@@ -0,0 +1,79 @@
+##===========SETUP===============##
+sudo yum install -y java-17-amazon-corretto-devel
+java -version
+sudo yum -y update
+sudo yum -y install docker
+docker -v
+sudo service docker start
+sudo yum -y install git
+sudo yum remove -y python3.7
+sudo yum update -y
+sudo yum groupinstall "Development Tools" -y
+sudo yum erase openssl-devel -y
+sudo yum install openssl11 openssl11-devel  libffi-devel bzip2-devel wget -y
+wget https://www.python.org/ftp/python/3.9.9/Python-3.9.9.tgz
+tar -xf Python-3.9.9.tgz
+cd Python-3.9.9/
+./configure --enable-optimizations
+make -j $(nproc)
+sudo make altinstall
+sudo yum install -y python3-pip
+cd ~
+sudo update-alternatives --install /usr/bin/python3 python3 /usr/local/bin/python3.9 1
+
+##===========OpenSearch===============##
+git clone https://github.com/opensearch-project/OpenSearch.git
+cd OpenSearch
+git checkout feature/extensions
+./gradlew clean
+./gradlew publishToMavenLocal
+./gradlew localDistro
```

Comment:
Why do we have to run `localDistro` here? Won't `assemble` work?

Reply:
Good point.  What exactly is localDistro?

Reply:
I used ./gradlew localDistro because it gives the folder extracted itself, ready to start the OpenSearch Cluster. If I do gradle assemble then I have to add extract commands for the tar file. Doing local Distro serves the purpose